pkg/proxy/proxy.go

```diff
@@ -99,6 +99,7 @@ type ProxySource interface {
 	GetLabels() []string
 	GetLabelsSHA() string
 	GetIdentity() policy.NumericIdentity
+	GetIdentityFromConsumable(policy.NumericIdentity) *policy.Identity
```

Comment:
`ResolveIdentity()`

Reply:
sure. Although I think I was following the Get* convention of  ProxySource interface

Reply:
done